docs/UserGuide.md

```diff
@@ -124,6 +124,7 @@ Format: `edit INDEX [n/NAME] [desc/DESCRIPTION] [date/DATE] [s/STATUS] [t/TAG]
 * Existing values will be updated to the input values.
 * When editing tags, the existing tags of the task will be removed i.e adding of tags is not cumulative.
 * You can remove all the task’s tags by typing `t/` without specifying any tags after it.
+* There are 3 Task statuses: `expired`, `uncompleted` and `completed`. However you can cannot directly modify task's status to `expired`
```

Comment:
Should we also add explanation for what each of the statuses mean? Also there's a space missing in line 132

Reply:
I mean completed and uncompleted is self-explanatory.

Reply:
Ok can